src/shared/components/palette/text-area/write-text-area-field.component.ts

```diff
@@ -13,4 +13,17 @@ export class WriteTextAreaFieldComponent extends AbstractFieldWriteComponent imp
   ngOnInit() {
     this.textareaControl = this.registerControl(new FormControl(this.caseField.value));
   }
+
+  isIE(): boolean {
```

Comment:
Lee recently wrote a BrowserService that wrapped up this logic. Please check with him if it has been merged, and if so replace this function with a call to the BrowserService, which you will need to inject via the constructor

Reply:
> Lee recently wrote a BrowserService that wrapped up this logic. Please check with him if it has been merged, and if so replace this function with a call to the BrowserService, which you will need to inject via the constructor

yep. Done

Reply:
> Lee recently wrote a BrowserService that wrapped up this logic. Please check with him if it has been merged, and if so replace this function with a call to the BrowserService, which you will need to inject via the constructor

Yep. Done now